snap7/client.py

```diff
@@ -649,8 +652,8 @@ def as_read_area(self, area: str, dbnumber: int, start: int, size: int, wordlen:
 
     def _prepare_as_write_area(self, area: str, data: bytearray) -> Tuple[int, Array]:
         if area not in snap7.types.areas.values():
-            raise NotImplementedError(f"{area} is not implemented in snap7.types")
-        if area == snap7.types.S7AreaTM:
+            raise Snap7Exception(f"{area} is not implemented in snap7.types")
```

Comment:
sorry, i think we should probably just get rid of this snap7exection. I think this is a value error.